Sources/OpenTelemetryApi/Trace/Span.swift

```diff
@@ -127,6 +127,16 @@ public enum SpanAttributeConstants: String {
     case httpRouteKey = "http.route"
 }
 
+extension Span {
+    public func end() {
+        context.scope?.close()
+    }
+
+    public func end(endOptions: EndSpanOptions) {
```

Comment:
why are you not taking the timestamp from the `EndSpanOptions`? Is it because it's just an abstract class implementation?

Reply:
Yes, its like an abstract class implementation, it has no notion of timestamp